frontend/composables/useAsyncEquipmentIds.ts

```diff
@@ -0,0 +1,53 @@
+import { isNonNullish, unique } from "remeda";
+
+import { GetOeeEquipmentTypesWithEquipmentIdsDocument } from "~/generated/graphql/operations";
+
+
+
+export default function () {
+  return useAsyncQuery(GetOeeEquipmentTypesWithEquipmentIdsDocument,
+    {},
+    "default",
+    { errorPolicy: "ignore" }
+    , {
+      transform: (typeRes) => {
+      /** All equipment with OEE `equipmentType`. */
+        const oeeEquipment = computed(() => unique(
```

Comment:
Now that you're using the `transform` function, none of this functionality should use reactive component. We should remove all of these `computed` calls